playwright/components/locators.ts

```diff
@@ -17,6 +17,7 @@ export const LABEL = '[data-element="label"]';
 export const COMMON_INPUT = ".common-input__";
 export const BUTTON = 'button[type="button"]';
 export const COMMON_INPUT_PREFIX = '[data-element="textbox-prefix"]';
+export const CHARACTER_COUNT = 'div[data-element="character-count"]';
```

Comment:
For this line and the one below, they are referring to the same thing`'div[data-element="character-count"]',` can we only have one const and delete the other one?